### PR TITLE
Update macos-prerequisites.md

### PR DESCRIPTION
**Pull request by danmoseley**
As per https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md

### CODE REVIEW

**Comment by danmosemsft** [comment]
cc @bartonjs 
https://github.com/dotnet/corefx/issues/39701#issuecomment-514377618

**Review by bartonjs — APPROVED** [review]

**Comment by bartonjs** [comment]
That was... easier than I imagined.  Thanks, Dan :smile:.

**Comment by danmosemsft** [comment]
I didn't even clone the repo 😄

File: docs/core/macos-prerequisites.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ For download links and more information, see [.NET Core 1.1 downloads](https://w
 
 .NET Core 3.0 Preview 3 is supported on the following versions of macOS:
```
Comment (mairaw) [inline docs/core/macos-prerequisites.md]: Can you also change this to not specify a version?

```suggestion
.NET Core 3.0 is supported on the following versions of macOS:
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @danmosemsft! Can you update ms.date to today's date as well in addition to my comment about the version?

**Comment by danmosemsft** [comment]
done

**Review by mairaw — APPROVED** [review]
Thanks!

**Pull request merged**